src/Integration/Binding/BindingWorkflow.cs

```diff
@@ -348,6 +348,19 @@ private void AbortWorkflow(IProgressController controller, CancellationToken tok
             return this.BindingProjects.Select(Language.ForProject).Distinct();
         }
 
+        private void InformAboutFilteredOutProjects(IEnumerable<Project> filteredOut)
+        {
+            List<Project> projects = filteredOut.ToList();
+            if (projects.Count == 0)
```

Comment:
You probably talking about clarity here, if there was projects.None() I'd agree that it's better, but NOT any is less readable than  count == 0 IMO.